DeAceroDLabs/marketplace-frontend
# [DMP-503] Carrito view

## What is done

[[DMP-503] Cart view](https://deacero.atlassian.net/browse/DMP-503)

## How is done

1. Created a new route to `/carrito`
2. Created a new context where the shopping cart items will be stored.
3. Changed the action of the shoppingCart button based on the items in the shoppingCart Context.

## How to test

1. Intall new dependencies `npm i`
2. Run `npm start`.
3. Login to the app.
4. Click on the shopping cart item.
5. You should be redirected to `/carrito` and see the "carrito" legend.
